generated-interfaces.go

```diff
@@ -1506,3 +1506,5 @@ type Worker interface {
 	WaitForEvent(event string, predicate ...interface{}) interface{}
 	ExpectEvent(event string, cb func() error, predicates ...interface{}) (interface{}, error)
 }
+
+//go:generate bash scripts/generate-api.sh
```

Comment:
I think it might be better to put it into e.g. into the playwright.go since when the generate-interfaces is deleted, it won't work anymore.

Reply:
Done!